manifests/g/Google/WorkspaceSync/4.3.53.0/Google.WorkspaceSync.installer.yaml

```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.7-2-4
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Google.WorkspaceSync
```

Comment:
Package has dependency on Microsoft Outlook
```
Action ended 18:31:51: InstallFinalize. Return value 3.
No supported version of Microsoft Outlook® found. Supported versions are Microsoft Outlook® 2003 SP3 (version 11.0.8169 or greater), Microsoft Outlook® 2007 SP2 (version 12.0.6504.5000 or greater), Microsoft Outlook® 2010, Microsoft Outlook® 2013, and Microsoft Outlook® 2016.
```

Reply:
Is it possible to list this as a dependency somehow? I'm still researching this. I was able to get --silent to work.